libraries/AP_HAL_Linux/HAL_Linux_Class.cpp

```diff
@@ -53,13 +53,7 @@ static I2CDriver i2cDriver2(2);
 #elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_BBBMINI
 static I2CDriver i2cDriver0(2);
 #elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_MINLURE
-static const std::vector<const char *> i2c_devpaths({
-    /* UEFI with lpss set to ACPI */
-    "platform/80860F41:05",
-    /* UEFI with lpss set to PCI */
-    "pci0000:00/0000:00:18.6",
-});
-static I2CDriver  i2cDriver0(i2c_devpaths);
+static I2CDriver  i2cDriver0({ "platform/80860F41:05", "pci0000:00/0000:00:18.6" });
```

Comment:
well... I guess then we should remove the path then.  Humn... I'll add back the comment since "not supporting on PCI anymore" is just a temporary thing and might help adding our next platforms.